src/Misc/layoutbin/en-US/strings.json

```diff
@@ -537,7 +537,8 @@
   "StopContainer": "Stop Container",
   "Success": "Succeeded: ",
   "SupportedRepositoryEndpointNotFound": "Unable to match any source repository endpoints with any of the supported source providers.",
-  "SupportedTaskHandlerNotFound": "A supported task execution handler was not found. This error usually means the task does not carry an implementation that is compatible with your current operating system. Contact the task author for more details.",
+  "SupportedTaskHandlerNotFoundLinux": "A supported task execution handler was not found. The task does not carry an implementation that is compatible with your current operating system '{0}'. Contact the task author for more details.",
```

Comment:
The current operating system is not capable of running this task.  That typically means the task was written for Windows only.  For example, written for Windows Desktop PowerShell.